Objects/codeobject.c

```diff
@@ -452,10 +452,15 @@ static PyObject *
 code_sizeof(PyCodeObject *co, void *unused)
 {
     Py_ssize_t res;
-
     res = _PyObject_SIZE(Py_TYPE(co));
+    _PyCodeObjectExtra *co_extra = (_PyCodeObjectExtra*) co->co_extra;
+
     if (co->co_cell2arg != NULL && co->co_cellvars != NULL)
         res += PyTuple_GET_SIZE(co->co_cellvars) * sizeof(Py_ssize_t);
+
+    if (co_extra != NULL)
+	res += co_extra->ce_size * sizeof(co_extra->ce_extras[0]);
```

Comment:
No tools for auto-formatting are used with CPython sources. You should follow PEP 7 and good sense.